src/Grpc.AspNetCore.Server/Internal/HttpContextServerCallContext.cs

```diff
@@ -263,8 +263,15 @@ private void DeadlineExceeded(object state)
 
             _status = new Status(StatusCode.DeadlineExceeded, "Deadline Exceeded");
 
-            // TODO(JamesNK): I believe this sends a RST_STREAM with INTERNAL_ERROR. Grpc.Core sends NO_ERROR
-            HttpContext.Abort();
+            try
+            {
+                // TODO(JamesNK): I believe this sends a RST_STREAM with INTERNAL_ERROR. Grpc.Core sends NO_ERROR
+                HttpContext.Abort();
+            }
+            catch (Exception ex)
+            {
+                Log.DeadlineCancellationError(_logger, ex);
```

Comment:
This doesn't seem like the right fix

Reply:
It doesn't fix why a NRE was thrown. But it at least stops the process exiting.

I want to figure out why Abort could throw in https://github.com/aspnet/AspNetCore/issues/9239